<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<title>Gregory W. Schwartz</title>
<meta name="generator" content="Org mode" />
<meta name="author" content="Gregory W. Schwartz" />
<style type="text/css">
 <!--/*--><![CDATA[/*><!--*/
  .title  { text-align: center;
             margin-bottom: .2em; }
  .subtitle { text-align: center;
              font-size: medium;
              font-weight: bold;
              margin-top:0; }
  .todo   { font-family: monospace; color: red; }
  .done   { font-family: monospace; color: green; }
  .priority { font-family: monospace; color: orange; }
  .tag    { background-color: #eee; font-family: monospace;
            padding: 2px; font-size: 80%; font-weight: normal; }
  .timestamp { color: #bebebe; }
  .timestamp-kwd { color: #5f9ea0; }
  .org-right  { margin-left: auto; margin-right: 0px;  text-align: right; }
  .org-left   { margin-left: 0px;  margin-right: auto; text-align: left; }
  .org-center { margin-left: auto; margin-right: auto; text-align: center; }
  .underline { text-decoration: underline; }
  #postamble p, #preamble p { font-size: 90%; margin: .2em; }
  p.verse { margin-left: 3%; }
  pre {
    border: 1px solid #ccc;
    box-shadow: 3px 3px 3px #eee;
    padding: 8pt;
    font-family: monospace;
    overflow: auto;
    margin: 1.2em;
  }
  pre.src {
    position: relative;
    overflow: visible;
    padding-top: 1.2em;
  }
  pre.src:before {
    display: none;
    position: absolute;
    background-color: white;
    top: -10px;
    right: 10px;
    padding: 3px;
    border: 1px solid black;
  }
  pre.src:hover:before { display: inline;}
  /* Languages per Org manual */
  pre.src-asymptote:before { content: 'Asymptote'; }
  pre.src-awk:before { content: 'Awk'; }
  pre.src-C:before { content: 'C'; }
  /* pre.src-C++ doesn't work in CSS */
  pre.src-clojure:before { content: 'Clojure'; }
  pre.src-css:before { content: 'CSS'; }
  pre.src-D:before { content: 'D'; }
  pre.src-ditaa:before { content: 'ditaa'; }
  pre.src-dot:before { content: 'Graphviz'; }
  pre.src-calc:before { content: 'Emacs Calc'; }
  pre.src-emacs-lisp:before { content: 'Emacs Lisp'; }
  pre.src-fortran:before { content: 'Fortran'; }
  pre.src-gnuplot:before { content: 'gnuplot'; }
  pre.src-haskell:before { content: 'Haskell'; }
  pre.src-hledger:before { content: 'hledger'; }
  pre.src-java:before { content: 'Java'; }
  pre.src-js:before { content: 'Javascript'; }
  pre.src-latex:before { content: 'LaTeX'; }
  pre.src-ledger:before { content: 'Ledger'; }
  pre.src-lisp:before { content: 'Lisp'; }
  pre.src-lilypond:before { content: 'Lilypond'; }
  pre.src-lua:before { content: 'Lua'; }
  pre.src-matlab:before { content: 'MATLAB'; }
  pre.src-mscgen:before { content: 'Mscgen'; }
  pre.src-ocaml:before { content: 'Objective Caml'; }
  pre.src-octave:before { content: 'Octave'; }
  pre.src-org:before { content: 'Org mode'; }
  pre.src-oz:before { content: 'OZ'; }
  pre.src-plantuml:before { content: 'Plantuml'; }
  pre.src-processing:before { content: 'Processing.js'; }
  pre.src-python:before { content: 'Python'; }
  pre.src-R:before { content: 'R'; }
  pre.src-ruby:before { content: 'Ruby'; }
  pre.src-sass:before { content: 'Sass'; }
  pre.src-scheme:before { content: 'Scheme'; }
  pre.src-screen:before { content: 'Gnu Screen'; }
  pre.src-sed:before { content: 'Sed'; }
  pre.src-sh:before { content: 'shell'; }
  pre.src-sql:before { content: 'SQL'; }
  pre.src-sqlite:before { content: 'SQLite'; }
  /* additional languages in org.el's org-babel-load-languages alist */
  pre.src-forth:before { content: 'Forth'; }
  pre.src-io:before { content: 'IO'; }
  pre.src-J:before { content: 'J'; }
  pre.src-makefile:before { content: 'Makefile'; }
  pre.src-maxima:before { content: 'Maxima'; }
  pre.src-perl:before { content: 'Perl'; }
  pre.src-picolisp:before { content: 'Pico Lisp'; }
  pre.src-scala:before { content: 'Scala'; }
  pre.src-shell:before { content: 'Shell Script'; }
  pre.src-ebnf2ps:before { content: 'ebfn2ps'; }
  /* additional language identifiers per "defun org-babel-execute"
       in ob-*.el */
  pre.src-cpp:before  { content: 'C++'; }
  pre.src-abc:before  { content: 'ABC'; }
  pre.src-coq:before  { content: 'Coq'; }
  pre.src-groovy:before  { content: 'Groovy'; }
  /* additional language identifiers from org-babel-shell-names in
     ob-shell.el: ob-shell is the only babel language using a lambda to put
     the execution function name together. */
  pre.src-bash:before  { content: 'bash'; }
  pre.src-csh:before  { content: 'csh'; }
  pre.src-ash:before  { content: 'ash'; }
  pre.src-dash:before  { content: 'dash'; }
  pre.src-ksh:before  { content: 'ksh'; }
  pre.src-mksh:before  { content: 'mksh'; }
  pre.src-posh:before  { content: 'posh'; }
  /* Additional Emacs modes also supported by the LaTeX listings package */
  pre.src-ada:before { content: 'Ada'; }
  pre.src-asm:before { content: 'Assembler'; }
  pre.src-caml:before { content: 'Caml'; }
  pre.src-delphi:before { content: 'Delphi'; }
  pre.src-html:before { content: 'HTML'; }
  pre.src-idl:before { content: 'IDL'; }
  pre.src-mercury:before { content: 'Mercury'; }
  pre.src-metapost:before { content: 'MetaPost'; }
  pre.src-modula-2:before { content: 'Modula-2'; }
  pre.src-pascal:before { content: 'Pascal'; }
  pre.src-ps:before { content: 'PostScript'; }
  pre.src-prolog:before { content: 'Prolog'; }
  pre.src-simula:before { content: 'Simula'; }
  pre.src-tcl:before { content: 'tcl'; }
  pre.src-tex:before { content: 'TeX'; }
  pre.src-plain-tex:before { content: 'Plain TeX'; }
  pre.src-verilog:before { content: 'Verilog'; }
  pre.src-vhdl:before { content: 'VHDL'; }
  pre.src-xml:before { content: 'XML'; }
  pre.src-nxml:before { content: 'XML'; }
  /* add a generic configuration mode; LaTeX export needs an additional
     (add-to-list 'org-latex-listings-langs '(conf " ")) in .emacs */
  pre.src-conf:before { content: 'Configuration File'; }

  table { border-collapse:collapse; }
  caption.t-above { caption-side: top; }
  caption.t-bottom { caption-side: bottom; }
  td, th { vertical-align:top;  }
  th.org-right  { text-align: center;  }
  th.org-left   { text-align: center;   }
  th.org-center { text-align: center; }
  td.org-right  { text-align: right;  }
  td.org-left   { text-align: left;   }
  td.org-center { text-align: center; }
  dt { font-weight: bold; }
  .footpara { display: inline; }
  .footdef  { margin-bottom: 1em; }
  .figure { padding: 1em; }
  .figure p { text-align: center; }
  .equation-container {
    display: table;
    text-align: center;
    width: 100%;
  }
  .equation {
    vertical-align: middle;
  }
  .equation-label {
    display: table-cell;
    text-align: right;
    vertical-align: middle;
  }
  .inlinetask {
    padding: 10px;
    border: 2px solid gray;
    margin: 10px;
    background: #ffffcc;
  }
  #org-div-home-and-up
   { text-align: right; font-size: 70%; white-space: nowrap; }
  textarea { overflow-x: auto; }
  .linenr { font-size: smaller }
  .code-highlighted { background-color: #ffff00; }
  .org-info-js_info-navigation { border-style: none; }
  #org-info-js_console-label
    { font-size: 10px; font-weight: bold; white-space: nowrap; }
  .org-info-js_search-highlight
    { background-color: #ffff00; color: #000000; font-weight: bold; }
  .org-svg { width: 90%; }
  /*]]>*/-->
</style>
<link rel="stylesheet" type="text/css" href="styles/readtheorg/css/htmlize.css"/>
<link rel="stylesheet" type="text/css" href="styles/readtheorg/css/readtheorg.css"/>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
<script type="text/javascript" src="styles/lib/js/jquery.stickytableheaders.min.js"></script>
<script type="text/javascript" src="styles/readtheorg/js/readtheorg.js"></script>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.5.1/jquery.js"></script>
<script src="./canvaslife/canvaslife.js"></script>
<script type="text/javascript">
/*
@licstart  The following is the entire license notice for the
JavaScript code in this tag.

Copyright (C) 2012-2019 Free Software Foundation, Inc.

The JavaScript code in this tag is free software: you can
redistribute it and/or modify it under the terms of the GNU
General Public License (GNU GPL) as published by the Free Software
Foundation, either version 3 of the License, or (at your option)
any later version.  The code is distributed WITHOUT ANY WARRANTY;
without even the implied warranty of MERCHANTABILITY or FITNESS
FOR A PARTICULAR PURPOSE.  See the GNU GPL for more details.

As additional permission under GNU GPL version 3 section 7, you
may distribute non-source (e.g., minimized or compacted) forms of
that code without the copy of the GNU GPL normally required by
section 4, provided you include this license notice and a URL
through which recipients can access the Corresponding Source.


@licend  The above is the entire license notice
for the JavaScript code in this tag.
*/
<!--/*--><![CDATA[/*><!--*/
 function CodeHighlightOn(elem, id)
 {
   var target = document.getElementById(id);
   if(null != target) {
     elem.cacheClassElem = elem.className;
     elem.cacheClassTarget = target.className;
     target.className = "code-highlighted";
     elem.className   = "code-highlighted";
   }
 }
 function CodeHighlightOff(elem, id)
 {
   var target = document.getElementById(id);
   if(elem.cacheClassElem)
     elem.className = elem.cacheClassElem;
   if(elem.cacheClassTarget)
     target.className = elem.cacheClassTarget;
 }
/*]]>*///-->
</script>
<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        displayAlign: "center",
        displayIndent: "0em",

        "HTML-CSS": { scale: 100,
                        linebreaks: { automatic: "false" },
                        webFont: "TeX"
                       },
        SVG: {scale: 100,
              linebreaks: { automatic: "false" },
              font: "TeX"},
        NativeMML: {scale: 100},
        TeX: { equationNumbers: {autoNumber: "AMS"},
               MultLineWidth: "85%",
               TagSide: "right",
               TagIndent: ".8em"
             }
});
</script>
<script type="text/javascript"
        src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS_HTML"></script>
</head>
<body>
<div id="content">
<h1 class="title">Gregory W. Schwartz</h1>
<div id="table-of-contents">
<h2>Table of Contents</h2>
<div id="text-table-of-contents">
<ul>
<li><a href="#orga1f9257">Current</a></li>
<li><a href="#org97eabe9">Statement</a></li>
<li><a href="#orgcf78866">Recent work</a></li>
<li><a href="#orgaa3afde">Publications</a></li>
<li><a href="#org792dc7c">Partial list of programs and libraries</a></li>
<li><a href="#org5316047">Self-organization</a></li>
<li><a href="#org801be72">Contact</a></li>
</ul>
</div>
</div>

<div id="outline-container-orga1f9257" class="outline-2">
<h2 id="orga1f9257">Current</h2>
<div class="outline-text-2" id="text-orga1f9257">
<p>
I am a postdoctoral fellow in the <a href="http://faryabib.github.io/">Faryabi Lab</a> at the University of Pennsylvania.
</p>
</div>
</div>

<div id="outline-container-org97eabe9" class="outline-2">
<h2 id="org97eabe9">Statement</h2>
<div class="outline-text-2" id="text-org97eabe9">
<p>
Life is a complex system of interconnecting networks: from the transcriptome to
the proteome, from a single cell to clonal populations, each layer of
information can be interpreted as a network where each molecule, cell, or tissue
in some way affects the overall system. <b>Systems biology</b> attempts to take into
account as many of these levels as possible in order to answer biological
questions pertaining to not only the organization of life but also separating
signal from noise to discover key components of disease or adverse clinical
outcomes. While sampling technologies give partial insight to these issues, the
other half of the solution, namely mathematical and computational analyses of
biological data, are needed more than ever in this high-throughput era.
</p>

<p>
The need for such analyses drives my research. In my efforts to elucidate the
relationships between genotype and phenotype, clonal diversity and selection,
and integration of multiple modalities of information (transcriptome, proteome,
phosphoproteome, etc.), I have developed algorithms, programs, and mathematical
measures.
</p>

<p>
More specifically, I am interested in clonal behavior and evolution
(<a href="#orgdb9df8a">10</a>, <a href="#org7e78091">5</a>, <a href="#org125903c">8</a>, <a href="#org5cbc99d">3</a>), pattern
recognition of the interactome through integration (<a href="#orgc818585">4</a>), network
structure similarity, data visualization (<a href="#org125903c">8</a>), and everything
in-between.
</p>
</div>
</div>

<div id="outline-container-orgcf78866" class="outline-2">
<h2 id="orgcf78866">Recent work</h2>
<div class="outline-text-2" id="text-orgcf78866">

<div class="figure">
<p><object type="image/svg+xml" data="img/too-many-cells.svg" class="org-svg">
Sorry, your browser does not support SVG.</object>
</p>
</div>

<p>
Single-cell analyses enable measurement of the transcriptome, chromatin
accessibility, and more at single-cell resolution. However, grouping similar
cells at different resolution cutoffs severely limits the exploration of both
common and rare heterogeneous populations. We have developed <code>TooManyCells</code>, a
suite of novel tools focused on simultaneous visualization of single-cell clade
relationships at all resolutions (<a href="#orgcaaa811">1</a>).
</p>


<div class="figure">
<p><object type="image/svg+xml" data="img/heatitup.svg" class="org-svg">
Sorry, your browser does not support SVG.</object>
</p>
</div>

<p>
Poor prognosis of patients with acute myeloid leukemia can be predicted based on
the presence of internal tandem duplications (ITDs) in Fms-like tyrosine kinase
3 (<i>FLT3</i>). We discovered new classes of <i>FLT3</i>-ITDs that predict patient
outcomes by creating <code>HeatITup</code>, an algorithm that identifies, characterizes,
and visualizes these classes of <i>FLT3</i>-ITDs (<a href="#org5cbc99d">3</a>).
</p>


<div class="figure">
<p><object type="image/svg+xml" data="img/integreat.svg" class="org-svg">
Sorry, your browser does not support SVG.</object>
</p>
</div>

<p>
While we can measure different modalities, such as the transcriptome and
proteome, it is difficult to analytically relate each level of information in a
systematic manner. We developed <code>inteGREAT</code>, an algorithm to use integration and
differential integration between conditions to identify biomarkers across
modalities (<a href="#orgc818585">4</a>).
</p>
</div>
</div>

<div id="outline-container-orgaa3afde" class="outline-2">
<h2 id="orgaa3afde">Publications</h2>
<div class="outline-text-2" id="text-orgaa3afde">
<ul class="org-ul">
<li><a id="orgcaaa811"></a> <b>Schwartz GW</b>, Petrovic J, Fasolino M, Zhou Y, Cai S, Xu L,
Pear WS, Vahedi G, and Faryabi RB. TooManyCells identifies and visualizes
relationships of single-cell clades. <i>Nat. Methods</i> (In press).
<a href="https://www.biorxiv.org/content/early/2019/01/13/519660">bioRxiv: doi:10.1101/519660</a></li>
<li>Petrovic J, Zhou Y, Fasolino M, Goldman N, <b>Schwartz GW</b>, Mumbach MR, Nguyen SC,
Rome KS, Sela Y, Zapataro Z, Blacklow SC, Kruhlak MJ, Shi J, Aster JC, Joyce
EF, Little SC, Vahedi G, Pear WS, Faryabi RB. Oncogenic Notch Promotes
Long-Range Regulatory Interactions within Hyperconnected 3D Cliques. <i>Mol.
Cell.</i> <b>73</b> 1174-1190.e12. (2019).
<a href="https://www.cell.com/molecular-cell/fulltext/S1097-2765(19)30006-1?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1097276519300061%3Fshowall%3Dtrue">doi:10.1016/j.molcel.2019.01.006</a></li>
<li><a id="org5cbc99d"></a> <b>Schwartz GW</b>, Manning BS, Zhou Y, Velu PD,
Bigdeli A, Astles R, Lehman AW, Morrissette JJD, Perl AE, Li M, Carroll M, and
Faryabi RB. Classes of ITD predict outcomes in AML patients treated with FLT3
inhibitors. <i>Clin. Cancer Res.</i> (2019).
<a href="http://clincancerres.aacrjournals.org/content/25/2/573">doi:10.1158/1078-0432.CCR-18-0655</a></li>
<li><a id="orgc818585"></a> <b>Schwartz GW</b>, Petrovic J, Zhou Y, and Faryabi RB.
Differential integration of transcriptome and proteome identifies pan-cancer
prognostic biomarkers. <i>Front. Genet.</i> <b>9</b>, 205 (2018).
<a href="https://www.frontiersin.org/articles/10.3389/fgene.2018.00205/full">doi:10.3389/fgene.2018.00205</a></li>
<li><a id="org7e78091"></a> Meng W*, Zhang B*, <b>Schwartz GW</b>, Rosenfeld AM, Ren D, Thome J
JC, Carpenter DJ, Matsuoka N, Lerner H, Friedman AL, Granot T, Farber DL,
Shlomchik MJ, Hershberg U and Luning Prak ET. An atlas of B cell clonal
distribution in the human body. <i>Nat. Biotechnol.</i> <b>35</b>, 879-884 (2017).
<a href="https://www.nature.com/nbt/journal/vaop/ncurrent/full/nbt.3942.html">doi:10.1038/nbt.3942</a></li>
<li>Antell GC, Dampier W, Aiamkitsumrit B, Nonnemacher MR, Jacobson JM, Pirrone V,
Zhong W, Kercher K, and Passic S, Williams JW, <b>Schwartz G</b>, Hershberg U,
Krebs FC, and Wigdahl B. Utilization of HIV-1 envelope V3 to identify X4- and
R5-specific Tat and LTR sequence signatures. <i>Retrovirology</i> <b>13</b>, 32 (2016).
<a href="http://retrovirology.biomedcentral.com/articles/10.1186/s12977-016-0266-9">doi:10.1186/s12977-016-0266-9</a></li>
<li><b>Schwartz GW</b>, Shauli T, Linial M, and Hershberg U. Serine substitutions are
linked to codon usage and differ for variable and conserved protein regions.
<i>Sci. Rep.</i> <b>9</b>, 1 (2019).
<a href="http://www.nature.com/articles/s41598-019-53452-3">doi:10.1038/s41598-019-53452-3</a></li>
<li><a id="org125903c"></a> <b>Schwartz GW</b>, Shokoufandeh A, Ontañón S, and Hershberg U.
Using a novel clumpiness measure to unite data with metadata: finding common
sequence patterns in immune receptor germline V genes. <i>Pattern Recogn. Lett.</i>
<b>74</b>, 24-29 (2016). <a href="http://www.sciencedirect.com/science/article/pii/S0167865516000234">doi:10.1016/j.patrec.2016.01.011</a></li>
<li>Meng W, Jayaraman S, Zhang B, <b>Schwartz GW</b>, Daber RD, Hershberg U,
Garfall AL, Carlson CS and Luning Prak ET. Trials and tribulations with VH
replacement. <i>Front. Immunol.</i> <b>5</b>, 10 (2014). <a href="http://www.frontiersin.org/Journal/10.3389/fimmu.2014.00010/abstract">doi:10.3389/fimmu.2014.00010</a></li>
<li><a id="orgdb9df8a"></a> <b>Schwartz GW</b>, Hershberg U. Germline Amino Acid
Diversity in B Cell Receptors is a Good Predictor of Somatic Selection
Pressures. <i>Front. Immunol.</i> <b>4</b>, 357 (2013). <a href="http://www.frontiersin.org/Journal/10.3389/fimmu.2013.00357/abstract">doi:10.3389/fimmu.2013.00357</a></li>
<li><a id="org90f65ca"></a> <b>Schwartz GW</b>, Hershberg U. Conserved variation:
identifying patterns of stability and variability in BCR and TCR V genes with
different diversity and richness metrics. <i>Phys. Biol.</i> <b>10</b>, 035005 (2013).
<a href="http://iopscience.iop.org/1478-3975/10/3/035005/">doi:10.1088/1478-3975/10/3/035005</a></li>
</ul>
</div>
</div>

<div id="outline-container-org792dc7c" class="outline-2">
<h2 id="org792dc7c">Partial list of programs and libraries</h2>
<div class="outline-text-2" id="text-org792dc7c">
<div class="org-center">
<table border="2" cellspacing="0" cellpadding="6" rules="groups" frame="hsides">


<colgroup>
<col  class="org-left" />

<col  class="org-center" />

<col  class="org-center" />

<col  class="org-center" />

<col  class="org-center" />
</colgroup>
<thead>
<tr>
<th scope="col" class="org-left">Program</th>
<th scope="col" class="org-center"><a href="https://github.com/GregorySchwartz">Github</a></th>
<th scope="col" class="org-center"><a href="http://hackage.haskell.org/user/GregorySchwartz">Hackage</a></th>
<th scope="col" class="org-center"><a href="https://www.stackage.org/">Stackage</a></th>
<th scope="col" class="org-center"><a href="https://hub.docker.com/u/gregoryschwartz">Docker</a></th>
</tr>
</thead>
<tbody>
<tr>
<td class="org-left">birch-beer</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/birch-beer">\(\bullet\)</a></td>
<td class="org-center"><a href="http://hackage.haskell.org/package/birch-beer">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
<td class="org-center"><a href="https://hub.docker.com/r/gregoryschwartz/birch-beer">\(\bullet\)</a></td>
</tr>

<tr>
<td class="org-left">clumpiness (<a href="#org125903c">8</a>, <a href="#org7e78091">5</a>)</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/clumpiness">\(\bullet\)</a></td>
<td class="org-center"><a href="http://hackage.haskell.org/package/clumpiness">\(\bullet\)</a></td>
<td class="org-center"><a href="https://www.stackage.org/lts-8.20/package/clumpiness-0.17.0.0">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
</tr>

<tr>
<td class="org-left">convert-annotation</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/convert-annotation">\(\bullet\)</a></td>
<td class="org-center"><a href="http://hackage.haskell.org/package/convert-annotation">\(\bullet\)</a></td>
<td class="org-center"><a href="https://www.stackage.org/lts-8.20/package/convert-annotation-0.5.0.1">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
</tr>

<tr>
<td class="org-left">differential</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/differential">\(\bullet\)</a></td>
<td class="org-center"><a href="http://hackage.haskell.org/package/differential">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
<td class="org-center">&#xa0;</td>
</tr>

<tr>
<td class="org-left">diversity (<a href="#org90f65ca">11</a>,<a href="#orgdb9df8a">10</a>)</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/diversity">\(\bullet\)</a></td>
<td class="org-center"><a href="http://hackage.haskell.org/package/diversity">\(\bullet\)</a></td>
<td class="org-center"><a href="https://www.stackage.org/lts-8.20/package/diversity-0.8.0.2">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
</tr>

<tr>
<td class="org-left">fasta</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/fasta">\(\bullet\)</a></td>
<td class="org-center"><a href="http://hackage.haskell.org/package/fasta">\(\bullet\)</a></td>
<td class="org-center"><a href="https://www.stackage.org/lts-8.20/package/fasta-0.10.4.2">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
</tr>

<tr>
<td class="org-left">find-clumpiness (<a href="#org125903c">8</a>, <a href="#org7e78091">5</a>)</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/find-clumpiness">\(\bullet\)</a></td>
<td class="org-center"><a href="http://hackage.haskell.org/package/find-clumpiness">\(\bullet\)</a></td>
<td class="org-center"><a href="https://www.stackage.org/lts-9.0/package/find-clumpiness-0.2.1.3">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
</tr>

<tr>
<td class="org-left">heatitup (<a href="#org5cbc99d">3</a>)</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/heatitup">\(\bullet\)</a></td>
<td class="org-center"><a href="https://hackage.haskell.org/package/heatitup">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
<td class="org-center">&#xa0;</td>
</tr>

<tr>
<td class="org-left">heatitup-complete (<a href="#org5cbc99d">3</a>)</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/heatitup-complete">\(\bullet\)</a></td>
<td class="org-center"><a href="https://hackage.haskell.org/package/heatitup-complete">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
<td class="org-center">&#xa0;</td>
</tr>

<tr>
<td class="org-left">hierarchical-spectral-clustering</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/hierarchical-spectral-clustering">\(\bullet\)</a></td>
<td class="org-center"><a href="http://hackage.haskell.org/package/hierarchical-spectral-clustering">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
<td class="org-center">&#xa0;</td>
</tr>

<tr>
<td class="org-left">integreat (<a href="#orgc818585">4</a>)</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/integreat">\(\bullet\)</a></td>
<td class="org-center"><a href="http://hackage.haskell.org/package/integreat">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
<td class="org-center">&#xa0;</td>
</tr>

<tr>
<td class="org-left">modify-fasta</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/modify-fasta">\(\bullet\)</a></td>
<td class="org-center"><a href="http://hackage.haskell.org/package/modify-fasta">\(\bullet\)</a></td>
<td class="org-center"><a href="https://www.stackage.org/lts-8.20/package/modify-fasta-0.8.2.3">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
</tr>

<tr>
<td class="org-left">modularity</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/modularity">\(\bullet\)</a></td>
<td class="org-center"><a href="http://hackage.haskell.org/package/modularity">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
<td class="org-center">&#xa0;</td>
</tr>

<tr>
<td class="org-left">random-tree (<a href="#org125903c">8</a>)</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/random-tree">\(\bullet\)</a></td>
<td class="org-center"><a href="http://hackage.haskell.org/package/random-tree">\(\bullet\)</a></td>
<td class="org-center"><a href="https://www.stackage.org/lts-8.20/package/random-tree-0.6.0.5">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
</tr>

<tr>
<td class="org-left">rank-product</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/rank-product">\(\bullet\)</a></td>
<td class="org-center"><a href="http://hackage.haskell.org/package/rank-product">\(\bullet\)</a></td>
<td class="org-center"><a href="https://www.stackage.org/lts-8.20/package/rank-product-0.2.0.1">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
</tr>

<tr>
<td class="org-left">spectral-clustering</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/spectral-clustering">\(\bullet\)</a></td>
<td class="org-center"><a href="http://hackage.haskell.org/package/spectral-clustering">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
<td class="org-center">&#xa0;</td>
</tr>

<tr>
<td class="org-left">too-many-cells</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/too-many-cells">\(\bullet\)</a></td>
<td class="org-center"><a href="http://hackage.haskell.org/package/too-many-cells">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
<td class="org-center"><a href="https://hub.docker.com/r/gregoryschwartz/too-many-cells">\(\bullet\)</a></td>
</tr>

<tr>
<td class="org-left">tree-fun (<a href="#org125903c">8</a>)</td>
<td class="org-center"><a href="https://github.com/GregorySchwartz/tree-fun">\(\bullet\)</a></td>
<td class="org-center"><a href="http://hackage.haskell.org/package/tree-fun">\(\bullet\)</a></td>
<td class="org-center"><a href="https://www.stackage.org/lts-8.20/package/tree-fun-0.8.1.0">\(\bullet\)</a></td>
<td class="org-center">&#xa0;</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>

<div id="outline-container-org5316047" class="outline-2">
<h2 id="org5316047">Self-organization</h2>
<div class="outline-text-2" id="text-org5316047">
<p>
<a href="https://github.com/jpulgarin/canvaslife">https://github.com/jpulgarin/canvaslife</a>
</p>

<div>
<input type="button" id="start" value="Toggle Life" onclick="life.toggleLife();">
<input type="button" value="End Life" onclick="life.clear();">
</div>

<canvas id="universe" width="500%" height="500%">
<script type="text/javascript">
$('document').ready(function() {
    life.initUniverse('#universe');
});
</script>
</div>
</div>

<div id="outline-container-org801be72" class="outline-2">
<h2 id="org801be72">Contact</h2>
<div class="outline-text-2" id="text-org801be72">
<p>
<a href="mailto:gsch@mail.med.upenn.edu">gsch@pennmedicine.upenn.edu</a>
</p>

<p>
<a href="https://www.linkedin.com/in/schwartzgregory">linkedin</a>
</p>

<p>
<a href="https://github.com/GregorySchwartz">github</a>
</p>

<p>
<a href="https://gitlab.com/GregorySchwartz">gitlab</a>
</p>

<p>
<a href="http://faryabib.github.io/">faryabi lab</a>
</p>

<p>
<a href="http://simlab.biomed.drexel.edu/">SImLab</a>
</p>
</div>
</div>
</div>
<div id="postamble" class="status">
<p class="author">Author: Gregory W. Schwartz</p>
<p class="email">Email: <a href="mailto:gsch@pennmedicine.upenn.edu">gsch@pennmedicine.upenn.edu</a></p>
<p class="validation"><a href="http://validator.w3.org/check?uri=referer">Validate</a></p>
</div>
</body>
</html>
