#+OPTIONS: ':nil *:t -:t ::t <:t H:3 \n:nil ^:t arch:headline author:t
#+OPTIONS: broken-links:nil c:nil creator:nil d:(not "LOGBOOK") date:nil e:t
#+OPTIONS: email:t f:t inline:t num:nil p:nil pri:nil prop:nil stat:t tags:t
#+OPTIONS: tasks:t tex:t timestamp:nil title:t toc:t todo:t |:t
#+TITLE: Gregory W. Schwartz
#+DATE: 
#+AUTHOR: Gregory W. Schwartz
#+EMAIL: gsch@pennmedicine.upenn.edu
#+LANGUAGE: en
#+SELECT_TAGS: export
#+EXCLUDE_TAGS: noexport
#+CREATOR:

#+SETUPFILE: ./setup/theme-readtheorg-local.setup

#+HTML_HEAD: <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.5.1/jquery.js"></script>
#+HTML_HEAD: <script src="./canvaslife/canvaslife.js"></script>

* Current
I am a Scientist at the Princess Margaret Cancer Centre and an Assistant
Professor at the University of Toronto in the Department of Medical Biophysics.

* Statement

Life is a complex system of interconnecting networks: from the transcriptome to
the proteome, from a single cell to clonal populations, each layer of
information can be interpreted as a network where each molecule, cell, or tissue
in some way affects the overall system. *Systems biology* attempts to take into
account as many of these levels as possible in order to answer biological
questions pertaining to not only the organization of life but also separating
signal from noise to discover key components of disease or adverse clinical
outcomes. While sampling technologies give partial insight to these issues, the
other half of the solution, namely mathematical and computational analyses of
biological data, are needed more than ever in this high-throughput era.

The need for such analyses drives my research. In my efforts to elucidate the
relationships between genotype and phenotype, clonal diversity and selection,
and integration of multiple modalities of information (transcriptome, proteome,
phosphoproteome, etc.), I have developed algorithms, programs, and mathematical
measures.

More specifically, I am interested in clonal behavior and evolution
([[diversitySelectionPaper]], [[bCellAtlas]], [[clumpinessPaper]], [[flt3Class]]), pattern
recognition of the interactome through integration ([[integration]]), network
structure similarity, data visualization ([[clumpinessPaper]]), and everything
in-between.

* Recent work

[[file:img/too-many-peaks_graphical_abstract.svg]]

While single-cell RNA sequencing can reveal much about the transcriptomic state
of individual cells, these states are largely set by the interaction of
transcription factors with regulatory elements within accessible chromatin
regions. To understand the relationship between cells with similar or unique
chromatin states, we developed =TooManyPeaks=, part of the =TooManyCells= suite,
for end-to-end analysis of single-cell assay for transposase-accessible
chromatin using sequencing (scATAC-seq) data ([[tooManyPeaks]]).

[[file:img/too-many-cells.svg]]

Single-cell analyses enable measurement of the transcriptome, chromatin
accessibility, and more at single-cell resolution. However, grouping similar
cells at different resolution cutoffs severely limits the exploration of both
common and rare heterogeneous populations. We have developed =TooManyCells=, a
suite of novel tools focused on simultaneous visualization of single-cell clade
relationships at all resolutions ([[tooManyCells]]).

[[file:img/heatitup.svg]]

Poor prognosis of patients with acute myeloid leukemia can be predicted based on
the presence of internal tandem duplications (ITDs) in Fms-like tyrosine kinase
3 (/FLT3/). We discovered new classes of /FLT3/-ITDs that predict patient
outcomes by creating =HeatITup=, an algorithm that identifies, characterizes,
and visualizes these classes of /FLT3/-ITDs ([[flt3Class]]).

[[file:img/integreat.svg]]

While we can measure different modalities, such as the transcriptome and
proteome, it is difficult to analytically relate each level of information in a
systematic manner. We developed =inteGREAT=, an algorithm to use integration and
differential integration between conditions to identify biomarkers across
modalities ([[integration]]).

* Publications

- <<tooManyPeaks>> *Schwartz GW*, Zhou Y, Petrovic J, and Faryabi RB. TooManyPeaks
    identifies drug-resistant-specific regulatory elements from single-cell
    leukemic epigenomes. /Cell Rep. *36*, (2021).
    [[https://doi.org/10.1016/j.celrep.2021.109575][doi:10.1016/j.celrep.2021.109575]]
- <<tooManyCells>> *Schwartz GW*, Petrovic J, Fasolino M, Zhou Y, Cai S, Xu L,
  Pear WS, Vahedi G, and Faryabi RB. TooManyCells identifies and visualizes
  relationships of single-cell clades. /Nat. Methods/ *17* 405–413 (2020).
  [[https://doi.org/10.1038/s41592-020-0748-5][doi:10.1038/s41592-020-0748-5]]
- Rome KS, Stein, SJ, Kurachi M, Petrovic J, *Schwartz GW*, Mack EA, Uljon S, Wu
  WW, DeHart AG, McClory SE, Xu L, Gimotty PA, Blacklow SC, Faryabi RB, Wherry
  EJ, Jordan MS, Pear WS. Trib1 regulates T cell differentiation during chronic
  infection by restraining the effector program. /J. Exp. Med./ *217* e20190888
  (2020).
  [[https://doi.org/10.1084/jem.20190888][doi:10.1084/jem.20190888]]
- Petrovic J, Zhou Y, Fasolino M, Goldman N, *Schwartz GW*, Mumbach MR, Nguyen SC,
  Rome KS, Sela Y, Zapataro Z, Blacklow SC, Kruhlak MJ, Shi J, Aster JC, Joyce
  EF, Little SC, Vahedi G, Pear WS, Faryabi RB. Oncogenic Notch Promotes
  Long-Range Regulatory Interactions within Hyperconnected 3D Cliques. /Mol.
  Cell./ *73* 1174-1190.e12. (2019).
  [[https://www.cell.com/molecular-cell/fulltext/S1097-2765(19)30006-1?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1097276519300061%3Fshowall%3Dtrue][doi:10.1016/j.molcel.2019.01.006]]
- <<flt3Class>> *Schwartz GW*, Manning BS, Zhou Y, Velu PD,
  Bigdeli A, Astles R, Lehman AW, Morrissette JJD, Perl AE, Li M, Carroll M, and
  Faryabi RB. Classes of ITD predict outcomes in AML patients treated with FLT3
  inhibitors. /Clin. Cancer Res./ (2019).
  [[http://clincancerres.aacrjournals.org/content/25/2/573][doi:10.1158/1078-0432.CCR-18-0655]]
  Commentary: Prognostic Models Turn the Heat(IT)up on FLT3^{ITD}-Mutated AML.
  [[http://clincancerres.aacrjournals.org/content/25/2/460?iss=2][doi:10.1158/1078-0432.CCR-18-3146]]
- <<integration>> *Schwartz GW*, Petrovic J, Zhou Y, and Faryabi RB.
  Differential integration of transcriptome and proteome identifies pan-cancer
  prognostic biomarkers. /Front. Genet./ *9*, 205 (2018).
  [[https://www.frontiersin.org/articles/10.3389/fgene.2018.00205/full][doi:10.3389/fgene.2018.00205]]
- <<bCellAtlas>> Meng W*, Zhang B*, *Schwartz GW*, Rosenfeld AM, Ren D, Thome J
  JC, Carpenter DJ, Matsuoka N, Lerner H, Friedman AL, Granot T, Farber DL,
  Shlomchik MJ, Hershberg U and Luning Prak ET. An atlas of B cell clonal
  distribution in the human body. /Nat. Biotechnol./ *35*, 879-884 (2017).
  [[https://www.nature.com/nbt/journal/vaop/ncurrent/full/nbt.3942.html][doi:10.1038/nbt.3942]]
- Antell GC, Dampier W, Aiamkitsumrit B, Nonnemacher MR, Jacobson JM, Pirrone V,
  Zhong W, Kercher K, and Passic S, Williams JW, *Schwartz G*, Hershberg U,
  Krebs FC, and Wigdahl B. Utilization of HIV-1 envelope V3 to identify X4- and
  R5-specific Tat and LTR sequence signatures. /Retrovirology/ *13*, 32 (2016).
  [[http://retrovirology.biomedcentral.com/articles/10.1186/s12977-016-0266-9][doi:10.1186/s12977-016-0266-9]]
- *Schwartz GW*, Shauli T, Linial M, and Hershberg U. Serine substitutions are
  linked to codon usage and differ for variable and conserved protein regions.
  /Sci. Rep./ *9*, 1 (2019).
  [[http://www.nature.com/articles/s41598-019-53452-3][doi:10.1038/s41598-019-53452-3]]
- <<clumpinessPaper>> *Schwartz GW*, Shokoufandeh A, Ontañón S, and Hershberg U.
  Using a novel clumpiness measure to unite data with metadata: finding common
  sequence patterns in immune receptor germline V genes. /Pattern Recogn. Lett./
  *74*, 24-29 (2016). [[http://www.sciencedirect.com/science/article/pii/S0167865516000234][doi:10.1016/j.patrec.2016.01.011]]
- Meng W, Jayaraman S, Zhang B, *Schwartz GW*, Daber RD, Hershberg U,
  Garfall AL, Carlson CS and Luning Prak ET. Trials and tribulations with VH
  replacement. /Front. Immunol./ *5*, 10 (2014). [[http://www.frontiersin.org/Journal/10.3389/fimmu.2014.00010/abstract][doi:10.3389/fimmu.2014.00010]]
- <<diversitySelectionPaper>> *Schwartz GW*, Hershberg U. Germline Amino Acid
  Diversity in B Cell Receptors is a Good Predictor of Somatic Selection
  Pressures. /Front. Immunol./ *4*, 357 (2013). [[http://www.frontiersin.org/Journal/10.3389/fimmu.2013.00357/abstract][doi:10.3389/fimmu.2013.00357]]
- <<diversityPaper>> *Schwartz GW*, Hershberg U. Conserved variation:
  identifying patterns of stability and variability in BCR and TCR V genes with
  different diversity and richness metrics. /Phys. Biol./ *10*, 035005 (2013).
  [[http://iopscience.iop.org/1478-3975/10/3/035005/][doi:10.1088/1478-3975/10/3/035005]]

* Partial list of programs and libraries
#+BEGIN_CENTER
| <l>                                                |     <c>     |     <c>     |     <c>     |     <c>     |
| Program                                            |   [[https://github.com/GregorySchwartz][Github]]    |   [[http://hackage.haskell.org/user/GregorySchwartz][Hackage]]   |  [[https://www.stackage.org/][Stackage]]   |   [[https://hub.docker.com/u/gregoryschwartz][Docker]]    |
|----------------------------------------------------+-------------+-------------+-------------+-------------|
| birch-beer                                         | [[https://github.com/GregorySchwartz/birch-beer][\(\bullet\)]] | [[http://hackage.haskell.org/package/birch-beer][\(\bullet\)]] |             | [[https://hub.docker.com/r/gregoryschwartz/birch-beer][\(\bullet\)]] |
| clumpiness ([[clumpinessPaper]], [[bCellAtlas]])           | [[https://github.com/GregorySchwartz/clumpiness][\(\bullet\)]] | [[http://hackage.haskell.org/package/clumpiness][\(\bullet\)]] | [[https://www.stackage.org/lts-8.20/package/clumpiness-0.17.0.0][\(\bullet\)]] |             |
| convert-annotation                                 | [[https://github.com/GregorySchwartz/convert-annotation][\(\bullet\)]] | [[http://hackage.haskell.org/package/convert-annotation][\(\bullet\)]] | [[https://www.stackage.org/lts-8.20/package/convert-annotation-0.5.0.1][\(\bullet\)]] |             |
| differential                                       | [[https://github.com/GregorySchwartz/differential][\(\bullet\)]] | [[http://hackage.haskell.org/package/differential][\(\bullet\)]] |             |             |
| diversity ([[diversityPaper]],[[diversitySelectionPaper]]) | [[https://github.com/GregorySchwartz/diversity][\(\bullet\)]] | [[http://hackage.haskell.org/package/diversity][\(\bullet\)]] | [[https://www.stackage.org/lts-8.20/package/diversity-0.8.0.2][\(\bullet\)]] |             |
| fasta                                              | [[https://github.com/GregorySchwartz/fasta][\(\bullet\)]] | [[http://hackage.haskell.org/package/fasta][\(\bullet\)]] | [[https://www.stackage.org/lts-8.20/package/fasta-0.10.4.2][\(\bullet\)]] |             |
| find-clumpiness ([[clumpinessPaper]], [[bCellAtlas]])      | [[https://github.com/GregorySchwartz/find-clumpiness][\(\bullet\)]] | [[http://hackage.haskell.org/package/find-clumpiness][\(\bullet\)]] | [[https://www.stackage.org/lts-9.0/package/find-clumpiness-0.2.1.3][\(\bullet\)]] |             |
| heatitup ([[flt3Class]])                               | [[https://github.com/GregorySchwartz/heatitup][\(\bullet\)]] | [[https://hackage.haskell.org/package/heatitup][\(\bullet\)]] |             |             |
| heatitup-complete ([[flt3Class]])                      | [[https://github.com/GregorySchwartz/heatitup-complete][\(\bullet\)]] | [[https://hackage.haskell.org/package/heatitup-complete][\(\bullet\)]] |             |             |
| hierarchical-spectral-clustering                   | [[https://github.com/GregorySchwartz/hierarchical-spectral-clustering][\(\bullet\)]] | [[http://hackage.haskell.org/package/hierarchical-spectral-clustering][\(\bullet\)]] |             |             |
| integreat ([[integration]])                            | [[https://github.com/GregorySchwartz/integreat][\(\bullet\)]] | [[http://hackage.haskell.org/package/integreat][\(\bullet\)]] |             |             |
| modify-fasta                                       | [[https://github.com/GregorySchwartz/modify-fasta][\(\bullet\)]] | [[http://hackage.haskell.org/package/modify-fasta][\(\bullet\)]] | [[https://www.stackage.org/lts-8.20/package/modify-fasta-0.8.2.3][\(\bullet\)]] |             |
| modularity                                         | [[https://github.com/GregorySchwartz/modularity][\(\bullet\)]] | [[http://hackage.haskell.org/package/modularity][\(\bullet\)]] |             |             |
| random-tree ([[clumpinessPaper]])                      | [[https://github.com/GregorySchwartz/random-tree][\(\bullet\)]] | [[http://hackage.haskell.org/package/random-tree][\(\bullet\)]] | [[https://www.stackage.org/lts-8.20/package/random-tree-0.6.0.5][\(\bullet\)]] |             |
| rank-product                                       | [[https://github.com/GregorySchwartz/rank-product][\(\bullet\)]] | [[http://hackage.haskell.org/package/rank-product][\(\bullet\)]] | [[https://www.stackage.org/lts-8.20/package/rank-product-0.2.0.1][\(\bullet\)]] |             |
| spectral-clustering                                | [[https://github.com/GregorySchwartz/spectral-clustering][\(\bullet\)]] | [[http://hackage.haskell.org/package/spectral-clustering][\(\bullet\)]] |             |             |
| too-many-cells                                     | [[https://github.com/GregorySchwartz/too-many-cells][\(\bullet\)]] | [[http://hackage.haskell.org/package/too-many-cells][\(\bullet\)]] |             | [[https://hub.docker.com/r/gregoryschwartz/too-many-cells][\(\bullet\)]] |
| tree-fun ([[clumpinessPaper]])                         | [[https://github.com/GregorySchwartz/tree-fun][\(\bullet\)]] | [[http://hackage.haskell.org/package/tree-fun][\(\bullet\)]] | [[https://www.stackage.org/lts-8.20/package/tree-fun-0.8.1.0][\(\bullet\)]] |             |
#+END_CENTER

* Self-organization

[[https://github.com/jpulgarin/canvaslife]]

#+BEGIN_EXPORT html
<div>
<input type="button" id="start" value="Toggle Life" onclick="life.toggleLife();">
<input type="button" value="End Life" onclick="life.clear();">
</div>

<canvas id="universe" width="500%" height="500%">
<script type="text/javascript">
$('document').ready(function() {
    life.initUniverse('#universe');
});
</script>
#+END_EXPORT

* Contact

[[mailto:gregory.schwartz@uhnresearch.ca][gregory.schwartz@uhnresearch.ca]]

[[https://www.linkedin.com/in/schwartzgregory][linkedin]]

[[https://github.com/GregorySchwartz][github]]
